### DISCUSSION TITLE
Valid of y+ in wind simulation

### DISCUSSION THREAD

**Discussion by togersen** [firemodels/fds#13857] [Q&A]
Hi

I have some simple simulations of an square bulding with the dimension of 10x10x10 with different cell size of 10 - 40 cm around the bulding. 

I have made some monitor points for velocity, pressure end the pressure coefficient. It seems like the values might convert with a cell size of 10 cm. But if I look and the y+ value it is not between 30-100. If I reduce cell size to 2.5 cm it is still not fulfilled. So what do you guys do when making cfd simulations with wind in fds regarding the y+ value?

**Comment by mcgratta** [comment]
Assuming that you are using the default `SIMULATION_MODE`, you are relying on empirical correlations of surface friction; i.e. a kind of "wall model", to estimate the drag on the flow. At the resolutions you cite, you are not resolving the boundary layer directly, so I would suggest you rely on a grid sensitivity study, as you have done, rather than some recommended range of y+. In my experience with the CFD literature and "best practices", whatever you do is never enough. Your grid is never fine enough. I prefer a grid sensitivity study.

**Reply by togersen** [reply]
In the model surfaces are modelled as inert surfaces. The wind is modelled with a constant value and no roughness are used in the model. So I think it should med normal simulation mode. 

I agree that a mesh sensitivity analyze is more benificial then only looking at the y+ value
